src/components/Medium/index.js

```diff
@@ -43,6 +43,23 @@ function Medium() {
   };
 
   const BlogCard = ({ blog }) => {
+    const getFirstParagraph = (blogContent) => {
+      let firstParagraph = null;
+      let div = document.createElement("div");
+      div.innerHTML = blogContent;
+      let elements = div.childNodes;
+
+      // Find the first paragraph element in the blog content
+      for (let i = 0; i < elements.length; i++) {
+        if (elements[i].nodeName === "P") {
+          firstParagraph = elements[i].innerText;
```

Comment:
Using `innerText` as opposed to `innerHTML` to avoid the need of using `dangerouslySetInnerHTML`. Although HTML elements in the description would no longer be injected into the DOM (ie. `<a>`,` <strong>`, etc.), I think the text alone should suffice in providing the user with enough info. These element styles and/or CTA are still valid if they choose to visit the blog on Medium.